components/motor/dmc4000/dmc.go

```diff
@@ -500,6 +500,8 @@ func (m *Motor) stopJog() error {
 // revolutions at a given speed in revolutions per minute. Both the RPM and the revolutions
 // can be assigned negative values to move in a backwards direction. Note: if both are
 // negative the motor will spin in the forward direction.
+// If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
+// revolutions may not be 0.
```

Comment:
I don't think we need to say revolutions may not be 0, because in other cases I don't think we say what _isn't_ allowed. like how we don't say that rpm can't be 0 for the same functions